packages/components/text/package.json

```diff
@@ -43,7 +43,8 @@
   },
   "peerDependencies": {
     "react": "17.0.2",
-    "react-native": "^0.68.0"
+    "react-native": "^0.68.0",
+    "@fluentui-react-native/interactive-hooks": ">=0.19.0 <1.0.0"
```

Comment:
I don't think this is the proper way to handle this, because this isn't what peer dependencies are meant for. Peer dependencies are for us to declare compatibility with a different library, and generally they don't get automatically installed, so it's not a good way to declare an actual dependency. This is why we have RN as both a dev and a peer dependency.

I think the proper solution would be to move createIconProps into a different package so that interactive-hooks doesn't depend on the icon package. And I think that change makes sense anyway because createIconProps isn't a hook or something that's for interactivity.

Reply:
I moved it to the icon package for now since it's a hook that will always be used when IconProps are defined or an Icon slot is created.